src/Mvc/src/Microsoft.AspNetCore.Mvc.TagHelpers/DisplayTagHelper.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Mvc.ViewFeatures.Buffers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers
+{
+    [HtmlTargetElement("display", Attributes = "for", TagStructure = TagStructure.WithoutEndTag)]
+    public class DisplayTagHelper : TagHelper
```

Comment:
This will end up in a separate PR, but I wanted to make sure scenarios work E2E. We'd give these a treatment similar to `PartialTagHelper` and encourage users to avoid using the blocking `HtmlHelper.Editor` \ `HtmlHelper.Display` apis